### PR TITLE
Improvements to /plot list command

### PR DESCRIPTION
**Pull request by Xaver106**
# Seperated from #3584

## Overview
Some improvements to `/p list`

## Description
- Shows all possible versions of the owner special: Own message for when the owner is the Server (server Flag), when the owner is everyone and when the owner is unknown (UUID invalid/not in cache)
- Plot shows now owned color if plot is owned by everyone

## Submitter Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] I tested my changes and approved their functionality.
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: Core/src/main/resources/lang/messages_en.json [hunk 1]
```diff
@@ -381,6 +381,9 @@
   "info.plot_list_default": "<gold><plot></gold>",
   "info.plot_list_player_online": "<dark_aqua><prefix></dark_aqua><hover:show_text:\"<dark_aqua>Online</dark_aqua>\"><gold><player></gold></hover>",
   "info.plot_list_player_offline": "<dark_aqua><prefix></dark_aqua><hover:show_text:\"<dark_gray>Offline</dark_gray>\"><gold><player></gold></hover>",
+  "info.plot_list_player_unknown": "<hover:show_text:\"<red>The owner of this plot is unknown</red>\"><white><player></white></hover>",
+  "info.plot_list_player_server": "<hover:show_text:\"<red>The plot is owned by the server</red>\"><white><player></white></hover>",
+  "info.plot_list_player_everyone": "<hover:show_text:\"<blue>The plot is owned by everyone</blue>\"><white><player></white></hover>",
```
Comment (dordsor21) [inline Core/src/main/resources/lang/messages_en.json:386]: Why does the player placeholder need to be present here? The lack of a space is also not ideal if it does need to be there for whatever reason

Comment (RedstoneFuture) [reply Core/src/main/resources/lang/messages_en.json:386]: It is the same scheme of the existing two entries above. It shows the plot list with the plot-owners.

Comment (dordsor21) [reply Core/src/main/resources/lang/messages_en.json:386]: The comment is RE all three, and they would result in "The owner of the plot is unknownunknown" for example

Comment (Xaver106) [reply Core/src/main/resources/lang/messages_en.json:386]: The player Placeholder is so the Name is consistent with the corresponding names out of the box (`info.unknown`,`info.server` and `info.everyone`).

Comment (dordsor21) [reply Core/src/main/resources/lang/messages_en.json]: It is still duplicating the "unknown"/"server"/"everyone". You have those provided by the string itself and the placeholder

Comment (Xaver106) [reply Core/src/main/resources/lang/messages_en.json]: In what way is it duplicating?

Comment (Xaver106) [reply Core/src/main/resources/lang/messages_en.json]: image

Comment (dordsor21) [reply Core/src/main/resources/lang/messages_en.json]: AH I see it's a hover... Somehow missed that each time...

**Review by dordsor21 — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks for your contribution, we'll merge the PR in ~24h if there's no negative feedback.

**Pull request merged**